src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -205,6 +268,28 @@ describe('AuthenticatedAPIClient request interceptor', () => {
   });
 });
 
+describe('AuthenticatedAPIClient ensureCsrfToken request interceptor', () => {
+  [
+    [false, 'POST', false, {}, expectGetCsrfTokenToHaveBeenCalled, true],
```

Comment:
Question: Is this how this is typically implemented?  I imagine an object with named fields would make it simpler to understand these different cases since it is separated from the actual function.

Reply:
I don't really love this either, but I'm not really sure how you would use an object here to make it better. Maybe a possible improvement to keep in mind when you start writing frontend tests.